php_memcached.stub.php

```diff
@@ -35,16 +35,16 @@ public function cas(string $cas_token, string $key, mixed $value, int $expiratio
 	public function casByKey(string $cas_token, string $server_key, string $key, mixed $value, int $expiration=0): bool {}
 	public function add(string $key, mixed $value, int $expiration=0): bool {}
 	public function addByKey(string $server_key, string $key, mixed $value, int $expiration=0): bool {}
-	public function append(string $key, string $value): bool {}
-	public function appendByKey(string $server_key, string $key, string $value): bool {}
-	public function prepend(string $key, string $value): bool {}
-	public function prependByKey(string $server_key, string $key, string $value): bool {}
+	public function append(string $key, string $value): ?bool {}
+	public function appendByKey(string $server_key, string $key, string $value): ?bool {}
+	public function prepend(string $key, string $value): ?bool {}
+	public function prependByKey(string $server_key, string $key, string $value): ?bool {}
```

Comment:
From `php_memc_store_impl` these method return NULL on error (cannot append/prepend with compression turned on) so another way is to raise an exception (PHP 8 only) in such case, as part of the "promote error to exception" global change.